### PR TITLE
Derryk/feature/viewfavoritebeans

### PR DESCRIPTION
**Pull request by DerrykBBD**
created and tested view fav beans

### CODE REVIEW

File: Server/src/main/java/com/bbd/BeanServer/service/FavoriteBeanService.java
Comment (jhviljoen) [inline Server/src/main/java/com/bbd/BeanServer/service/FavoriteBeanService.java:1]: For future reference, you should be able to use the _@Autowired_  annotation along with the repository variable. This saves some boilerplate, as it does the init by itself.

File: Server/src/main/java/com/bbd/BeanServer/model/FavoriteBean.java
Comment (jhviljoen) [inline Server/src/main/java/com/bbd/BeanServer/model/FavoriteBean.java:1]: I'm accepting this one, but just have a look a bit later, there's an annotation that auto-generates the constructor for us which I'll be adding here.

**Review by jhviljoen — APPROVED** [review]
All good.

File: Server/src/main/java/com/bbd/BeanServer/model/FavoriteBean.java
Comment (DerrykBBD) [reply Server/src/main/java/com/bbd/BeanServer/model/FavoriteBean.java:1]: I see adding it now , looking at docs.

**Pull request merged**